gatsby-node.ts

```diff
@@ -35,7 +35,7 @@ const getTitle = (frontmatter: any, lang?: any, db?: any) => {
     const titleEntry = frontmatter.techMetaTitles
       ? frontmatter.techMetaTitles.find((item: any) => item.name === queryParam)
       : null
-    pageSeoTitle = titleEntry ? titleEntry.value : `${pageSeoTitle} | ${queryParam}`
+    pageSeoTitle = titleEntry ? titleEntry.value : `${pageSeoTitle} : ${queryParam}`
```

Comment:
I'm not sure if this is quite correct 😄 
The current title contains `typescript-postgresql` from `query-param` I think:

```
Start from scratch with relational databases (15 min) : typescript-postgresql
```

Reply:
This seems to be the current behaviour. You can go ahead and ignore the comment 😅